tempto-examples/src/main/resources/tempto-configuration.yaml

```diff
@@ -84,3 +84,9 @@ list:
     - ala
     - ma
     - kota
+
+ldap:
```

Comment:
would be nice to add some example (which to some extend serve as tests for tempto) which uses ldap.
This is some work to be done though as docker machine with ldap would be needed. Decide if it is worth the work.